packages/metrics/src/features/audit/handler.ts

```diff
@@ -27,9 +27,14 @@ export async function newAuditHandler(
   try {
     const remoteAddress =
       request.headers['x-real-ip'] || request.info.remoteAddress
+
     const userAgent =
       request.headers['x-real-user-agent'] || request.headers['user-agent']
+
     const payload = request.payload as IUserAuditBody
+
+    const transactionId = payload.transactionId
```

Comment:
Transaction IDs are used to determine if an operation has already been performed. A service can for instance check if there is already a database row with a certain ID. If yes, then it wont write it again